packages/devtools_app/lib/src/shared/primitives/utils.dart

```diff
@@ -1061,9 +1061,20 @@ extension StringExtension on String {
     return true;
   }
 
-  /// Whether [other] is a case insensitive match for this String
-  bool caseInsensitiveEquals(String? other) {
-    return toLowerCase() == other?.toLowerCase();
+  /// Whether [other] is a case insensitive match for this String.
+  ///
+  /// If [pattern] is a [RegExp], this method will return true iff this String
```

Comment:
nit: if (typo)

Reply:
I wrote iff to mean if and only iff. I will expand to if and only if for clarity.